### PR TITLE
typo: hander -> handler

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: xml/System.Windows.Input/ExecutedRoutedEventArgs.xml [hunk 1]
```diff
@@ -76,8 +76,8 @@
 ## Examples  
  The following example creates an <xref:System.Windows.Input.ExecutedRoutedEventHandler> that handles multiple commands. The handler checks the <xref:System.Windows.Input.ExecutedRoutedEventArgs.Command%2A> property on the <xref:System.Windows.Input.ExecutedRoutedEventArgs> to determine which method to call.  
   
- [!code-csharp[commandingoverviewsnippets#CommandingOverviewMultipleCmdHander](~/samples/snippets/csharp/VS_Snippets_Wpf/CommandingOverviewSnippets/CSharp/Window1.xaml.cs#commandingoverviewmultiplecmdhander)]
- [!code-vb[commandingoverviewsnippets#CommandingOverviewMultipleCmdHander](~/samples/snippets/visualbasic/VS_Snippets_Wpf/CommandingOverviewSnippets/visualbasic/window1.xaml.vb#commandingoverviewmultiplecmdhander)]  
+ [!code-csharp[commandingoverviewsnippets#CommandingOverviewMultipleCmdHandler](~/samples/snippets/csharp/VS_Snippets_Wpf/CommandingOverviewSnippets/CSharp/Window1.xaml.cs#commandingoverviewmultiplecmdhandler)]
```
Comment (nschonni) [inline xml/System.Windows.Input/ExecutedRoutedEventArgs.xml]: Not sure about these ones

Comment (mairaw) [reply xml/System.Windows.Input/ExecutedRoutedEventArgs.xml]: You'd have to change the name of the snippet tag here first for this to work

Comment (mairaw) [reply xml/System.Windows.Input/ExecutedRoutedEventArgs.xml]: Like here: https://github.com/dotnet/samples/blob/master/snippets/csharp/VS_Snippets_Wpf/CommandingOverviewSnippets/CSharp/Window1.xaml.cs#L192

File: xml/System.Windows.Input/ExecutedRoutedEventArgs.xml [hunk 2]
```diff
@@ -76,8 +76,8 @@
...
+ [!code-vb[commandingoverviewsnippets#CommandingOverviewMultipleCmdHandler](~/samples/snippets/visualbasic/VS_Snippets_Wpf/CommandingOverviewSnippets/visualbasic/window1.xaml.vb#commandingoverviewmultiplecmdhandler)]
```
Comment (mairaw) [inline xml/System.Windows.Input/ExecutedRoutedEventArgs.xml]: and here: https://github.com/dotnet/samples/blob/master/snippets/visualbasic/VS_Snippets_Wpf/CommandingOverviewSnippets/visualbasic/window1.xaml.vb#L167

**Comment by mairaw** [comment]
@nschonni can I also ask you. Is it possible to combine a bit of these changes into a larger PR?

**Pull request merged**